webapp/components/survey/CategoryDetails/ImportSummary/TableRow.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+import * as Category from '@core/survey/category'
+import * as CategoryImportSummary from '@core/survey/categoryImportSummary'
+import * as Languages from '@core/app/languages'
+
+import { useI18n } from '@webapp/store/system'
+import Dropdown from '@webapp/components/form/Dropdown'
+
+const TableRow = ({ idx, columnName, column, setCategoryImportSummaryColumnDataType }) => {
+  const i18n = useI18n()
+
+  const columnSummaryKey = CategoryImportSummary.hasColumnLang(column)
+    ? 'categoryEdit.importSummary.columnTypeSummaryWithLanguage'
+    : CategoryImportSummary.isColumnExtra(column)
+    ? 'categoryEdit.importSummary.columnTypeSummaryExtra'
+    : 'categoryEdit.importSummary.columnTypeSummary'
```

Comment:
does eslint allow this condition?

Reply:
no! the components in ImportSummary will be refactored in the next pull requests

Reply:
ok!